mita/util.py

```diff
@@ -6,7 +6,9 @@
 
 from mita.connections import jenkins_connection
 from mita.exceptions import CloudNodeNotFound
-from mita import expressions, providers
+from mita import providers
+
+from label_eval import matching_nodes
```

Comment:
it would be preferable to not import `label_eval` like this because if there is ever a top level module called the same (unlikely but still possible) it will cause unexpected behavior. Keeping the same style of imports as above will also keep things consistent and explicit as to where they come from:

    from mita.label_eval import maching_nodes